src/openapi/core/OpenAPI.ts

```diff
@@ -26,8 +26,5 @@ export const OpenAPI: OpenAPIConfig = {
   TOKEN: undefined,
   USERNAME: undefined,
   PASSWORD: undefined,
-  HEADERS: {
-    Authorization: process.env.NEXT_PUBLIC_API_TOKEN ?? '',
```

Comment:
Hmm is the autorization token not needed for connecting to the API or is this handled elsewhere?

Reply:
The token is no longer needed at the moment. Having this header actually breaks the requests. (Check out the demo branch where Conduction did the same)

Reply:
Hmmm don't know if it is related, but changing the date from the 26th to something else actually causes an `unauthorised` error <img width="435" alt="Screenshot 2023-04-26 at 14 18 30" src="https://user-images.githubusercontent.com/877246/234572800-17a6c45c-d1ed-4fd3-bd69-bbb7cb85d7d7.png">

Reply:
Hmmm weird, will look into it more this afternoon 👀

Reply:
But I see the requests in Vercel indeed do not have a authorization token in the header... 🤷